scripts/md2html/build.sh

```diff
@@ -2,22 +2,38 @@
 
 # Author: @MikeRalphson
 
-# run this script from the root of the repo. It is designed to be run by a GitHub workflow.
+# run this script from the root of the repo
+# It is designed to be run by a GitHub workflow
+
+# Usage: build.sh [version | "latest" | "oas"]
+# When run with no arguments, it builds artifacts for all published specification versions.
+# It may also be run with a specific version argument, such as "3.1.1" or "latest"
+# Finally, it may be run with "oas" to build "src/oas.md"
+#
 # It contains bashisms
 
-mkdir -p deploy/oas
-mkdir -p deploy/js
+if [ -z "$1" ]; then
+  deploydir="deploy/oas"
+else
+  deploydir="deploy-preview"
+fi
+
+mkdir -p $deploydir/js
+mkdir -p $deploydir/temp
+
+latest=$(git describe --abbrev=0 --tags)
+latestCopied="none"
+lastMinor="-"
 
-cd scripts/md2html
 mkdir -p history
```

Comment:
The whole "history" acrobatics has been removed on `main`, and unfortunately not yet merged into `dev` yet. The editors are now in sibling files to each specification version.

Reply:
I cherry-picked 10c10bc (materialize maintainers lists) to this PR and updated the build script to use the materialized versions.